scraper.py

```diff
@@ -285,3 +285,37 @@ def source(self):
         )
 
         return response.json()
+
+
+class Fragbite(Scraper):
+    """
+    Fragbite implements a reader for fragbite.se
+    News title contains number of comments inside "()"
+    """
+    @classmethod
+    def url(cls):
+        """
+        The URL for the site.
+        """
+        return "https://fragbite.se"
+
+    def headers(self):
+        """
+        Return a list of all headers for the site.
+        """
+        soup = BeautifulSoup(self.source(), "html.parser")
+
+        result = []
+
+        for article in soup.find_all("div", class_="text"):
+            subheading = article.find("div", class_="subheading")
+
+            result.append(
+                header.Header(
+                    article.h1.get_text(),
+                    subheading.get_text(),
+                    "{}{}".format(self.url(), article.a.get("href")),
+                    False,
+                )
+            )
+        return result
```

Comment:
Mind adding a newline for consistency?
```suggestion
            )
            
        return result
```

Reply:
Done!